modules/claims_api/app/controllers/claims_api/v1/forms/base.rb

```diff
@@ -52,7 +52,7 @@ def documents
           @documents ||= params.slice(*document_keys).values.map do |document|
             case document
             when String
-              decode_document(document)
+              document.blank? ? nil : decode_document(document)
```

Comment:
rack-test was converting a nil attachment to a blank string, which was causing a failure down the road here